src/CodeStyle/VisualBasic/CodeFixes/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.vbproj

```diff
@@ -7,7 +7,31 @@
     <OutputType>Library</OutputType>
     <TargetFramework>netstandard1.3</TargetFramework>
     <RootNamespace></RootNamespace>
+
+    <!-- NuGet -->
+    <IsPackable>true</IsPackable>
+    <IsAnalyzer>true</IsAnalyzer>
+    <NuspecPackageId>Microsoft.CodeAnalysis.VisualBasic.CodeStyle</NuspecPackageId>
+    <PackageDescription>
+      .NET Compiler Platform ("Roslyn") code style analyzers for Visual Basic.
+    </PackageDescription>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);_GetFilesToPackage</TargetsForTfmSpecificContentInPackage>
```

Comment:
📝 This doesn't follow the normal `xxxxxxDependsOn` pattern for properties referencing targets by name. If this is a property defined somewhere in this pull request, it might make sense to update it.

Reply:
This is NuGet extensibility point. It's not a list of dependent targets. Rather it's a list of targets that populate `TfmSpecificPackageFile` item group.